src/hevc.h

```diff
@@ -0,0 +1,31 @@
+/*
+    FLVMeta - FLV Metadata Editor
+
+    Copyright (C) 2007-2019 Marc Noirot <marc.noirot AT gmail.com>
+
+    This file is part of FLVMeta.
+
+    FLVMeta is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    FLVMeta is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with FLVMeta; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+*/
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
```

Comment:
Are all these include directives needed here?
I didn't try to build this PR, but I tried to remove `#include <stdio.h>` from avc.h and it builds, so I guess these might be unnecessary.
If they are needed in the hevc.c code, please put them there.